### PR TITLE
delete slash at the end of the url

### PR DESCRIPTION
**Pull request by SimonLab**
see my second comment on the [issue 3](https://github.com/akitten/linkedin-canonical-url/issues/3)
Sounds good to you @nelsonic ?

### CODE REVIEW

**Comment by SimonLab** [comment]
forget to update the version, I'm doing it now

**Comment by nelsonic** [comment]
Did you see my reply: https://github.com/akitten/linkedin-canonical-url/issues/3#issuecomment-162603066 ?

File: lib/index.js [hunk 1]
```diff
@@ -14,10 +14,13 @@ function transform (url) {
   url = url.split('?')[0];              // remove any query parameters
   parts = url.split('/');               // split url into parts
 
+  if(url.lastIndexOf('/') === url.length - 1) {
```
Comment (nelsonic) [inline lib/index.js]: I would add a comment here:

``` js
if(url.lastIndexOf('/') === url.length - 1) { // strip trailing slash from url
```

Comment (SimonLab) [inline lib/index.js]: :+1:

**Comment by nelsonic** [comment]
Cool. :+1:

**Comment by nelsonic** [comment]
New version on NPM. :+1:

**Pull request merged**